src/components/socialAppsLoginSignUp.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+    <div>
+        <v-btn icon class="mr-2" @click="createAccountwithFacebook">
+            <v-icon color="blue">mdi-facebook</v-icon>
+          </v-btn>
+          <v-btn icon class="mr-2" @click="createAccountwithGoogle">
+            <v-icon color="red">mdi-google</v-icon>
+          </v-btn>
+          <v-btn icon @click="createAccountwithTwitter">
+            <v-icon color="lightblue">mdi-twitter</v-icon>
+          </v-btn>
```

Comment:
use a json for all these button
json can have action,color,icon....whatever you needed @ShivanshSinghnnovative